.github/workflows/eslint.yml

```
@@ -0,0 +1,25 @@
+name: Lint
+
+on: 
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Our primary branch is `main`
```suggestion
    branches: [ main ]
  pull_request:
    branches: [ main ]
```